python/spacewalk/satellite_tools/repo_plugins/deb_src.py

```diff
@@ -511,6 +511,11 @@ def set_download_parameters(self, params, relative_path, target_file, checksum_t
         params["minrate"] = self.minrate
         params['proxies'] = get_proxies(self.repo.proxy, self.repo.proxy_username,
                                         self.repo.proxy_password)
+        with cfg_component('server.satellite') as CFG:
+            if CFG.urlgrabber_logspec:
+                params['urlgrabber_logspec'] = CFG.urlgrabber_logspec
+            else:
+                params['urlgrabber_logspec'] = None
```

Comment:
If you want to put explicit check if the option is present in the config, it's better to do it this way:
```suggestion
        params['urlgrabber_logspec'] = None
        with cfg_component('server.satellite') as CFG:
            if hasattr(CFG, "urlgrabber_logspec"):
                params["urlgrabber_logspec"] = CFG.urlgrabber_logspec
```
It seems there is no any proplem if the value is emptry, so it should be safe and no need to check if the value is not blank.